_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,144 @@
+
+You can update this setting in the source index's detail page as seen in the following image.
+![Image: Shrink-move-shards]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-move-shards.jpg){:.img-fluid }
+
+Or you can submit the following request with `Dev Tools`:
+
+```
+PUT test-1/_settings
+{
+"index.routing.allocation.require._name":"node1"
+}
+```
+
+After ensuring that all prerequisites are met, you can fill out the input form on the **Shrink index** page. Specify a name, the primary shard count, and the replica count for the new shrunken index as seen in the following image. You can also specify new aliases or select existing aliases to attach to the new shrunken index.
+
+![Image: Shrink-configure]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-configure.jpg){:.img-fluid }
+
+All of the settings specified above will also be shown in the JSON editor under the **Advanced settings** section. You can also specify additional index settings for the new shrunken index in the JSON editor. For example, you can set both `index.routing.allocation.require._name` and `index.blocks.write` to `null` as seen in the following image. This will clear the allocation requirement and the index write block, which are copied from the source index.
+
+
+![Image: Shrink-advanced-settings]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-advanced-settings.jpg){:.img-fluid }
+
+Once you have specified all of the settings for the new shrunken index, choose the **Shrink** button to trigger the shrink operation. You will see a notification indicating that the shrink operation has started successfully as seen in the following image.
+
+![Image: Shrink-started]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-started.jpg){:.img-fluid }
+
+After the shrink operation is completed, which may take anywhere from minutes to hours, a new notification will be displayed stating that the source index has been successfully shrunken as seen in the following image.
+
+![Image: Shrink-completed]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-completed.jpg){:.img-fluid }
+
+### Open and close index operation
+
+You can select multiple indexes, except for the backing indexes of a data stream, to open or close.
+
+If you no longer need to read or search old indexes but do not want to delete them, you can use the **Close** operation to close indexes. This will maintain the data, but will have a small overhead on the cluster. Additionally, when you want to add a new analyzer to an existing index, you must close the index, define the analyzer, and then open the index. A closed index is blocked for read and write operations, so you must type the word "close" to confirm your action as seen in the following image.
```

Comment:
```suggestion
If you no longer need to read or search old indexes but do not want to delete them, you can use the **Close** operation to close indexes. This will maintain the data but will have a small overhead on the cluster. Additionally, when you want to add a new analyzer to an existing index, you must close the index, define the analyzer, and then open the index. A closed index is blocked for read and write operations, so you must enter the word "close" to confirm your action, as shown in the following image.
```

Reply:
The meaning of "will have a small overhead" is unclear. Please rephrase to simplify/clarify.

Reply:
I like it, but do we mean "This will maintain the data by occupying a small amount of overhead on the cluster"? If yes, I've already made the change in my suggestion.